pkg/kubectl/resource_printer.go

```diff
@@ -1629,7 +1630,7 @@ func printEvent(event *api.Event, w io.Writer, options PrintOptions) error {
 
 // Sorts and prints the EventList in a human-friendly format.
 func printEventList(list *api.EventList, w io.Writer, options PrintOptions) error {
-	sort.Sort(SortableEvents(list.Items))
+	sort.Sort(events.SortableEvents(list.Items))
 	for i := range list.Items {
```

Comment:
Can someone help understand when is this function called . I was debugging this bug that I am trying to fix and my debugging reveals this function is not used at all. Also see this https://github.com/kubernetes/kubernetes/issues/29838